test.py

```diff
@@ -0,0 +1,64 @@
+from unittest import TestCase
+from io import BytesIO
+from os import path
+
+from PIL import Image as PilImage, ImageDraw as PilDraw
+from wand.image import Image as WandImage
+
+import dhash
+
+
+IMGDIR = path.dirname(__file__)
+
+
+def pilToWand(image, format='png'):
```

Comment:
Naming nit: let's pleas use `pil_to_wand` naming for this for consistency.

Reply:
yup, good to stick to this